doc/page-cmakefunctions.h

```diff
@@ -122,4 +122,17 @@ This function adds a test with the name \e TargetName.The parameter descriptions
 \param URI The URI of the Target i.e qml plugin
 \param VERSION The target i.e Qml plugin version, if not provided it defaults to 1.0
 
+\section cmake-params CMake options
+
+| Option                            | description                                                         |
+|-----------------------------------|---------------------------------------------------------------------|
+| FACELIFT_DISABLE_DBUS_IPC         | Disables IPC during build. Cannot use with FACELIFT_ENABLE_DBUS_IPC |
+| FACELIFT_ENABLE_CODEGEN           | generates code; should be used in rare cases only e.g. cross compilation of already generated code |
```

Comment:
This option enables the build and installation of the code generator. it does not "generate code". It can be useful to  set it to OFF when cross-compiling. mentioning "should be used in rare cases" has not value here. Just describe when it can be useful.